mandatory/4-tax.js

```diff
@@ -17,7 +21,11 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let finalPrice = (price * 1.2).toFixed(2);
```

Comment:
Would you be able to reuse the ``calculateSalesTax`` function that you wrote above here? Remember, always reuse the code. Never calculate or code things from scratch :)

Reply:
Yes, I would be able to, since I pass at least one parameter, the price.

Reply:
Have you done this? Not sure why it's resolved